pkg/distributor/distributor.go

```diff
@@ -273,6 +290,29 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help: "Unix timestamp of latest received sample per user.",
 		}, []string{"user"}),
 	}
+
+	promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "cortex_distributor_inflight_push_requests",
```

Comment:
In the ingester we have 1 metric exposing limits with the limit as label. Here we use a different pattern. I would keep the two consistent.